src/pages/gas.tsx

```diff
@@ -0,0 +1,493 @@
+              <Translation id="page-gas-what-causes-high-gas-fees-text-2" />
+            </Text>
+            <Text>
+              <Translation id="page-gas-what-causes-high-gas-fees-text-3" />
+            </Text>
+            <Text>
+              <Translation id="page-gas-want-to-dive-deeper" />{" "}
+              <InlineLink to="/developers/docs/gas/">
+                <Translation id="page-gas-check-out-the-developer-docs" />
+              </InlineLink>
+            </Text>
+          </Box>
+          <GhostCard
+            flex="40%"
+            maxW="640px"
+            alignSelf="center"
+            mt={{ base: 16, lg: 2 }}
+          >
+            <Emoji text=":cat:" fontSize="5xl" />
+            <H3>
+              <Translation id="page-gas-attack-of-the-cryptokitties-header" />
+            </H3>
+            <Text>
+              <Translation id="page-gas-attack-of-the-cryptokitties-text" />
+            </Text>
+          </GhostCard>
+        </Flex>
+      </Content>
+      <Content mb={{ base: 16, lg: 32 }}>
+        <Flex
+          direction={{ base: "column", lg: "row" }}
+          align="flex-start"
+          width="full"
+          mr={{ base: 0, lg: 8 }}
+        >
+          <Box w="full" mr={{ base: "auto", lg: "8" }}>
+            <Box>
+              <H2 mt={0}>
+                <Translation id="page-gas-why-do-we-need-gas-header" />
+              </H2>
+              <Text>
+                <Translation id="page-gas-why-do-we-need-gas-text" />
+              </Text>
+            </Box>
+            {benefits.map((benefit) => (
+              <Box key={benefit.description} minWidth="full" my={2}>
+                <HorizontalCard
+                  key={benefit.emoji}
+                  emoji={benefit.emoji}
+                  description={benefit.description}
+                  emojiSize={3}
+                  align="center"
+                />
+              </Box>
+            ))}
+          </Box>
+          <Box w="full">
+            <Image
+              src={ethImg}
+              alt=""
+              style={{ maxHeight: "400px" }}
+              objectFit="contain"
+            />
+          </Box>
+        </Flex>
+      </Content>
+      <Content mb={{ base: 16, lg: 32 }}>
+        <Flex direction={{ base: "column", lg: "row" }} align="flex-start">
+          <Box w="full" mr={{ base: "auto", lg: 8 }}>
+            <Flex alignItems="flex-start">
+              <H2 mt={0}>
+                <Translation id="page-gas-how-is-gas-calculated-header" />
+              </H2>
+
+              <Pill mt={1.5} ml={4} background="warning">
+                <Translation id="page-gas-advanced" />
+              </Pill>
+            </Flex>
+            <Text>
+              <Translation id="page-gas-how-is-gas-calculated-text-1" />
+            </Text>
+            <UnorderedList ml={6} spacing={3}>
+              <ListItem>
+                <Translation id="page-gas-how-is-gas-calculated-item-1" />
```

Comment:
Seeing six (6) i18n keys that correspond to values requiring `htmr` (and thus need the `Translation` component)... will flag these to add the `page-gas:` prefix, but the others can use the `t()` function with the namespace already loaded above

```suggestion
                <Translation id="page-gas:page-gas-how-is-gas-calculated-item-1" />
```